.gitignore

```diff
@@ -65,3 +67,4 @@ package-lock.json
 /ios/Pods/
 /ios/Podfile.lock
 expoDemo/ios/Pods
+
```

Comment:
gitignore should not be updated in this case.
gitignore file changes should be done intentionally and usually as a result of infra changes.

Reply:
@ArnonZ we added it on purpose. 
We notice that these files appear at one point when someone build the app for the first time. 
Not sure why. 
Anyway, I told Lidor to include it in the PR.

Reply:
Ok, might have been worth mentioning :)